docker/config/local/arronax/local/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "url": "https://conseil-local/",
+  "key": "hooman"
```

Comment:
Thought we moved away from `hooman` as the API key?

Reply:
this is arronax, I didn't touch arronax.

Reply:
Let's change it anyway or we'll leak our commonly-used API key.

Reply:
done

Reply:
Not done. Please double check your work.